app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
OK, I've set the presenter on the `app/controllers/concerns/sufia/files_controller_behavior.rb`